Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "StatsBase"
 uuid = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 authors = ["JuliaStats"]
-version = "0.32.0"
+version = "0.33.0"
```

Comment:
Is this really breaking? If it isn't breaking, it should `version = "0.32.1"` according to Pkg's semver (https://julialang.github.io/Pkg.jl/v1/toml-files/#The-version-field-1).